src/components/appointments/AppointmentProduct.stories.mdx

```diff
@@ -0,0 +1,31 @@
+import {Canvas, Meta, Story} from '@storybook/addon-docs';
+import {expect} from '@storybook/jest';
+import {within} from '@storybook/testing-library';
+
+import {FormikDecorator} from '../../story-utils/decorators';
+import AppointmentProduct from './AppointmentProduct';
+
+export const Template = args => <AppointmentProduct {...args} />;
+
+<Meta title="Private API / Appointments / Appointment Product" component={AppointmentProduct} />
+
+<Canvas>
+  <Story
+    name="Static"
+    decorators={[FormikDecorator]}
```

Comment:
I prefer the decorator being specified on the `<Meta>` component so it automatically applies to all stories.